crates/terminal/src/lib.rs

```diff
@@ -129,6 +129,24 @@ macro_rules! ceprint {
     };
 }
 
+pub fn print_link(url: &str) {
+    if atty::is(atty::Stream::Stdout) {
```

Comment:
Just a note that we should eventually move the std library support for this: https://doc.rust-lang.org/stable/std/io/trait.IsTerminal.html#tymethod.is_terminal